src/components/Dashboard/FilingHistoryList/subtitles/FutureEffectivePaid.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div class="future-effective-paid">
+    <span v-if="isTypeIncorporationApplication">FUTURE EFFECTIVE INCORPORATION</span>
+    <span v-else-if="isTypeAlteration">FUTURE EFFECTIVE ALTERATION</span>
+    <span v-else-if="isTypeDissolutionVoluntary">FUTURE EFFECTIVE DISSOLUTION</span>
+    <span v-else>FUTURE EFFECTIVE FILING</span>
+    <span class="vert-pipe" />
+    <span>PAID <FiledLabel :filing="filing" /></span>
```

Comment:
It seems like this component is called from a filing component that already "knows" the filing type.  What do you think of either passing in the title as a prop or as a slot with the default as "Future Effective Filing"?  This way could eliminate the need for this `if-then-else` logic.

Reply:
You're right. It's the same scenario as FutureEffective above.

Again, I left the existing code. My plan for this ticket was to clean up the bulk of the pasta in the FilingHistoryList template and script -- to separate filing types and to make it easier to add new ones. I wasn't planning on re-writing all sub-components (though I did many). How about a new ticket to handle this comment and the one above?